src/rpc/rpcevo.cpp

```diff
@@ -232,7 +267,7 @@ void protx_register_help()
 void protx_register_prepare_help()
 {
     throw std::runtime_error(
-            "protx register_prepare \"collateralHash\" collateralIndex \"ipAndPort\" \"ownerKeyAddr\" \"operatorPubKey\" \"votingKeyAddr\" operatorReward \"payoutAddress\"\n"
+            "protx register_prepare \"collateralHash\" collateralIndex \"ipAndPort\" \"ownerKeyAddr\" \"operatorPubKey\" \"votingKeyAddr\" operatorReward \"payoutAddress\" ( \"fundAddress\" )\n"
```

Comment:
`fundAddress` might be confusing here as it's not clear if this refers to the funds used for the collateral or for the fees. In this case (`protx register_prepare`), it's for the fees only, while in `protx register_fund` it's for the collateral and for the fees.

Maybe change the name of this parameter to `feeAddress` in all commands except `register_fund`?

Reply:
`feeAddress` sounds like "where to send fees" for me 🙈

Reply:
Maybe smth more general like `balanceAddress` or `sourceAddress`?

Reply:
done